packages/fx-core/src/core/index.ts

```diff
@@ -717,7 +717,7 @@ export class FxCore implements Core {
             description: "",
             author: "",
             scripts: {
-              test: "echo \"Error: no test specified\" && exit 1",
+              test: 'echo "Error: no test specified" && exit 1',
```

Comment:
per my previous observation, sometimes the code will be formatted to `"echo \"Error: no test specified\" && exit 1"` and sometimes it will be formatted to `'echo "Error: no test specified" && exit 1'`. Seems the behavior of code format command various in different machine?

Reply:
The code formatting will be done in this [task](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10851162)